### DISCUSSION TITLE
When does homebrew update its sources?

### DISCUSSION THREAD

**Discussion by nathanvy** [Homebrew/discussions#2963] [Everyday usage] [answered]
Reference:  https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/sbcl.rb

SBCL released version 2.2.1 on 30 January of this year.  The version available through homebrew, 2.1.9, is more than 4 months and two releases out of date.

Does homebrew depend on users submitting pull requests for packages when new versions are released?

**Comment by carlocab** [comment, marked as answer]
> Does homebrew depend on users submitting pull requests for packages when new versions are released?

Yes. However, in the case of sbcl: we just haven't been able to build the recently released versions. See [homebrew-core](https://github.com/Homebrew/homebrew-core/pulls?q=is%3Apr+in%3Atitle+sbcl) for recent attempts. Pull requests to update sbcl welcome.
